app/jobs/regular/pull_hotlinked_images.rb

```diff
@@ -104,15 +108,18 @@ def execute(args)
 
       end
 
+      post.custom_fields[LARGE_IMAGES] = large_images.to_s
```

Comment:
Why are you doing a `.to_s`? CustomField should support arrays. If they don't, it's best to serialize using JSON.